app/Fission/Environment/IPFS/Types.hs

```diff
@@ -5,17 +5,19 @@ import           Fission.Prelude
 import qualified Fission.IPFS.Types as IPFS
 
 data Environment = Environment
-  { url     :: !IPFS.URL     -- ^ IPFS client URL (may be remote)
-  , timeout :: !IPFS.Timeout -- ^ IPFS timeout in seconds
-  , binPath :: !IPFS.BinPath -- ^ Path to local IPFS binary
-  , gateway :: !IPFS.Gateway -- ^ Domain Name of IPFS Gateway
+  { url        :: !IPFS.URL     -- ^ IPFS client URL (may be remote)
+  , timeout    :: !IPFS.Timeout -- ^ IPFS timeout in seconds
+  , binPath    :: !IPFS.BinPath -- ^ Path to local IPFS binary
+  , gateway    :: !IPFS.Gateway -- ^ Domain Name of IPFS Gateway
+  , remotePeer :: !IPFS.Peer    -- ^ Remote Peer to connect to
   } deriving Show
 
 instance FromJSON Environment where
   parseJSON = withObject "IPFS.Environment" \obj -> do
-    timeout <- obj .:? "timeout" .!= 3600
-    binPath <- obj .:? "binPath" .!= "/usr/local/bin/ipfs"
-    gateway <- obj .:? "gateway" .!= "ipfs.runfission.com"
-    url     <- obj .:  "url" >>= parseJSON . String
+    timeout    <- obj .:? "timeout" .!= 3600
+    binPath    <- obj .:? "binPath" .!= "/usr/local/bin/ipfs"
+    gateway    <- obj .:? "gateway" .!= "ipfs.runfission.com"
+    url        <- obj .:  "url" >>= parseJSON . String
+    remotePeer <- obj .:  "remotePeer"
```

Comment:
should we default this to our peerID?

Reply:
So i'm not sure which way we want to fall on this.

Personally I'm uneasy about having default config values for things that could easily change (Like urls, ip addresses, certain types of keys). Simply because it lets us be less explicit in our configuration files. Which if these values change in the future we may be left having to read code to figure out how to override a default to get a production application back online.

@expede @dholms does that make sense?

Reply:
well the .env file would take precedence over in code values. So we could default `remotePeer` to `blahblahblah` and it would still work fine as long as we have an actual value in our .env file. This would just be for if the .env file is screwed up for some reason and the `remotePeer` value gets left out. Altho that _should_ never happen so it probably isn't a big deal

Reply:
I think I'm going to leave it without a default. I would hope that we could always look at the `env.yaml` and know what the application is configured to use.

Its mostly to guard against the following cases:
1. you wanted to change the `remotePeer` so you added the `remotePeer` to `env.yaml` however you mispelled it as `remotPeer`. Everything would run fine and you wouldn't know that the config change didnt occur.
2. we build a test environment, including a test s3node. You forget to override in local development and now your pinning test files to s3 production.

Holding off on merging till after lunch in case theres an over my dead body :)